test/list.js

```diff
@@ -3,19 +3,19 @@ import { assert } from 'chai';
 import * as RA from '../src';
 
 describe('list', function() {
-  it('tests creating list from items', function() {
+  it('creates list from items', function() {
```

Comment:
For this whole file here, we have a convention to always start a test description with a `should` word.

`creates list from another list` -> `should create list from another list`